content/developer/howtos/website_themes/theming.rst

```diff
@@ -0,0 +1,632 @@
+            'o-cc*-h6':                     'o-color-*',
+            'o-cc*-link':                   'o-color-*',
+            'o-cc*-btn-primary':            'o-color-*',
+            'o-cc*-btn-primary-border':     'o-color-*',
+            'o-cc*-btn-secondary':          'o-color-*',
+            'o-cc*-btn-secondary-border':   'o-color-*',
+
+         ),
+      )
+   );
+
+.. note::
+   For each `o-cc*`, replace the `*` with the digit (1 - 5) corresponding to the desired color
+   combination.
+
+.. note::
+   The default text color is `o-color-5`. If the background is too dark, it will automatically
+   change to the `o-color-4` color.
+
+.. seealso::
+   `Color combinations SCSS
+   <{GITHUB_PATH}/addons/web_editor/static/src/scss/web_editor.common.scss#L708>`_
+
+.. admonition:: Demo page
+
+   The Website Builder automatically generates a page to view the color combinations of the theme
+   color palette: `http://localhost:8069/website/demo/color-combinations`
+
+Bootstrap variables
+-------------------
+
+Odoo includes Bootstrap by default. You can use all variables and mixins of the framework.
+
+If Odoo does not provide the variable you are looking for, there could be a Bootstrap variable that
+allows it. Indeed all Odoo layouts respect Bootstrap structures and use Bootstrap components or
+their extensions. If you customize a Bootstrap variable, you add a generic style for the whole user
+website.
+
+Use a dedicated file added to the :file:`_assets_frontend_helpers` bundle to override Bootstrap
+values and *not* the :file:`primary_variables.scss` file.
+
+**Declaration**
+
+.. code-block:: python
+   :caption: ``/website_airproof/__manifest__.py``
+
+   'assets': {
+      'web._assets_frontend_helpers': [
+         ('prepend', 'website_airproof/static/src/scss/bootstrap_overridden.scss'),
+      ],
+   },
+
+**Use**
+
+.. code-block:: scss
+   :caption: ``/website_airproof/static/src/scss/bootstrap_overridden.scss``
+
+   // Typography
+   $h1-font-size:                 4rem !default;
+
+   // Navbar
+   $navbar-nav-link-padding-x:    1rem!default;
+
+   // Buttons + Forms
+   $input-placeholder-color:      o-color('o-color-1') !default;
+
+   // Cards
+   $card-border-width:            0 !default;
+
+.. tip::
+   That file must only contain definitions and overrides of SCSS variables and mixins.
+
+.. warning::
+   Don't override Bootstrap variables that depend on Odoo variables. Otherwise, you might break the
+   possibility for the user to customize them using the Website Builder.
+
+.. seealso::
+   `Bootstrap overridden SCSS
+   <{GITHUB_PATH}/addons/website/static/src/scss/bootstrap_overridden.scss>`_
+
+.. admonition:: Demo page
+
+   `http://localhost:8069/website/demo/bootstrap`
+
+Views
+-----
+
+For some options, in addition to the Website Builder variable, you also have to activate a specific
+view.
+
+By reading the source code, templates related to options are easily found.
```

Comment:
It's worth commenting that this can be also made with the `theme.utils` abstract model and the `_<theme_name>_post_copy` wildcard method using `enable_view`, `disable_view`, `enable_asset`, `disable_asset`.